src/main.js

```diff
@@ -27,6 +27,30 @@ async function init() {
   gameState.characterBody = characterBody;
   scene.add(hitboxMesh);
 
+  // Add a rotating cube
+   const geometry = new THREE.BoxGeometry();
+   const material = new THREE.MeshBasicMaterial({ color: 0x00ff00 });
+   const cube = new THREE.Mesh(geometry, material);
+   scene.add(cube);
+   camera.position.z = 5;
+
+  // Setup raycaster and mouse
+   const raycaster = new THREE.Raycaster();
+   const mouse = new THREE.Vector2();
+
+   function onDocumentClick(event) {
```

Comment:
this seems overkill see: https://dev.to/pahund/adding-a-click-handler-to-a-three-js-model-1f28